doc/source/serve/monitoring.md

```diff
@@ -233,15 +233,100 @@ In addition to the standard Python logger, Serve supports custom logging. Custom
-{"levelname": "INFO", "asctime": "2023-07-17 10:34:25,440", "deployment": "default_api", "replica": "default_api#bFDOnw", "request_id": "jLTczxOqme", "route": "/app1", "application": "default", "message": "replica.py:664 - Started executing request jLTczxOqme"}
-{"levelname": "INFO", "asctime": "2023-07-17 10:34:25,441", "deployment": "default_api", "replica": "default_api#bFDOnw", "request_id": "jLTczxOqme", "route": "/app1", "application": "default", "message": "replica.py:691 - __CALL__ OK 0.1ms"}
+### Configure Serve logging 
+From ray 2.9, logging_config API is introduced to configure logging for Ray Serve. You can configure logging for Ray Serve by passing a dictionary to `logging_config` argument of `serve.run` or `@serve.deployment`.
+
+#### Configure logging format
+You can configure json logging format by passing `encoding=JSON` to `logging_config` argument `@serve.deployment`. For example
+
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __json_start__
+:end-before: __json_end__
+:language: python
+```
+In the `Model` log file, you should see
+
+```
+{"levelname": "INFO", "asctime": "2023-12-07 12:59:45,271", "deployment": "Model", "replica": "default#Model#PUGBSJ", "request_id": "8d316c3b-4c9f-4769-8080-b6867ca46d1e", "route": "/", "application": "default", "message": "replica.py:719 - Started executing request 8d316c3b-4c9f-4769-8080-b6867ca46d1e"}
+{"levelname": "INFO", "asctime": "2023-12-07 12:59:45,271", "deployment": "Model", "replica": "default#Model#PUGBSJ", "request_id": "8d316c3b-4c9f-4769-8080-b6867ca46d1e", "route": "/", "application": "default", "message": "replica.py:745 - __CALL__ OK 0.1ms"}
+```
+
+#### Configure logging to a new file
+You can configure to use different directory for logging by passing `log_dir` to `logging_config` argument `@serve.deployment`. For example
+
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __logs_dir_start__
+:end-before:  __logs_dir_end__
+:language: python
+```
+
+The `Model` replica log file should be at `/my_dirs` directory.
+
+#### Configure logging level
+You can configure logging level by passing `log_level` to `logging_config` argument `@serve.deployment`. For example
+
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __level_start__
+:end-before:  __level_end__
+:language: python
+```
+
+The `Model` replica log file should also print `DEBUG` level message.
+
+```
+INFO 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default replica.py:719 - Started executing request f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1
+DEBUG 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default logging_config.py:26 - this is debug message
+INFO 2023-12-07 13:07:15,788 Model default#Model#LMqLpq f16c2ddb-9b4a-4a05-9b22-7d48f4ee51f1 / default replica.py:745 - __CALL__ OK 0.1ms
+```
+
+#### Disable access log
```

Comment:
Can we add a note above where we describe the access log that it can be disabled with this option?